deepcell_applications/utils_test.py

```diff
@@ -135,3 +139,77 @@ def test_get_predict_kwargs(mocker):
         bad_namespace = copy.copy(mock_namespace)
         del bad_namespace['test']
         _ = dca.utils.get_predict_kwargs(bad_namespace)
+
+
+def test_get_arg_parser():
+    with tempfile.TemporaryDirectory() as temp_dir:
+        img = np.zeros((10, 10))
```

Comment:
Maybe we should refactor this name to reflect that it really only tests mesmer. (`test_get_arg_parser_mesmer`?)

The pattern looks good, but because each subparser is so different, the dictionary will be totally different for the next app.